hw3/02-charts.js

```diff
@@ -29,17 +29,53 @@ const borderColors = [
 // url for the Thrones API
 const url = 'https://thronesapi.com/api/v2/Characters';
 
+const houseLabels = {
+  "House Targaryen": 0,
+  "House Tarly": 0,
+  "House Stark": 0,
+  "House Baratheon": 0,
+  "House Lannister": 0,
+  "House Greyjoy": 0,
+  "House Clegane": 0,
+  "House Baelish": 0,
+  "House Seaworth": 0,
+  "Unknown": 0,
+  "House Tyrell": 0,
+  "Other" : 0
+}
+
+let houses = []
+fetch(url)
+    .then(response => response.json())
+    .then(data => {
+        console.log(data);
```

Comment:
console.log messages such as this are useful for development and debugging, but they don't need to be merged into main.

Reply:
Oops! Removed.